client/local.cpp

```diff
@@ -284,36 +284,45 @@ int build_local(CompileJob &job, MsgChannel *local_daemon, struct rusage *used)
         child_pid = fork();
     }
 
-    if (!child_pid) {
+    if (!child_pid || child_pid == -1) {
```

Comment:
if `child_pid == -1` we need to log that. 

I'm not sure running the job is correct in the case where we cannot fork.